CMakeLists.txt

```
@@ -50,12 +50,12 @@ if ("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ggdb -DDALI_DEBUG=1")
```

Not a fan of introducing unrelated CMake changes in test framework implementation.

If we move -O2 to only release, I guess we can add 
```suggestion
  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Og -ggdb -g3 -DDALI_DEBUG=1")
```
here.

---

Done